helm/values/csi-driver/v1.1.0-beta/values_3parprimera.yaml

```diff
@@ -0,0 +1,58 @@
+# Default values for hpe-csi-storage.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+images:
+  # csi-driver
+  csiDriverImage: hpestorage/csi-driver:v1.1.0-beta
+  # csp-hpe
+  cspImage: dockerciuser/hpe_3par_primera_csp:test
+
+# image pull policy for all images in csi deployment
+imagePullPolicy: 'Always'
+
+# CSP deployment name
+cspName: hpe3parprimera-csp
+
+secret:
+  # parameters for hpe3parprimera
+  create: true
+  backend: 15.212.195.246
+  username: 3paradm
+  password: 3pardata
+  servicePort: "10000"
+  serviceName: hpe3parprimera-csp-svc
+  name: hpe3parprimera-secret
+
+# flavor
+flavor: kubernetes
+
+# log level for all csi driver components
+logLevel: info
+
+## For creating the StorageClass automatically:
+storageClass:
+  create: false
```

Comment:
Should we set this to true?

Reply:
I think in the sc we give the parameters related to product specific also the features which we support. so it will be better if we not include it in the deployment currently.